apps/react-workshop/src/Tree.stories.tsx

```diff
@@ -682,3 +682,122 @@ export const Small = () => {
     />
   );
 };
+
+export const VirtualizedWithHorizontalScroll = () => {
```

Comment:
Some changes to the story:
- Lets increase the max depth, because that's when horizontal scroll is most useful.
- Lets remove extra features like `sublabel`s and disabled nodes so that the code becomes simpler. These features are not relevant for virtualization or horizontal scroll.

Reply:
Removed the sublabels and disabled nodes, and changed how the sub items are generated so that every node has a deeper sub node tree in this [commit](https://github.com/iTwin/iTwinUI/pull/2170/commits/49e554d67de2d05f1a5cc4f02d286a27f446194c).